arch/x86_64/src/intel64/intel64_stackframe.c

```diff
@@ -85,7 +85,7 @@ void *up_stack_frame(struct tcb_s *tcb, size_t frame_size)
     }
 
   ret = tcb->stack_base_ptr;
-  memset(ret, 0, tcb->adj_stack_size);
+  memset(ret, 0, frame_size);
```

Comment:
@szafonimateusz-mi could you please include a commit log message explaining why it should be a fixed frame_size instead of tcb->adj_stack_size ?

Reply:
done